### PR TITLE
Make test for hdf5 serializer and deserializer

### PR DESCRIPTION
**Pull request by unnonouno**

### CODE REVIEW

File: tests/chainer_tests/serializers_tests/test_hdf5.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        self.assertEqual(serializer.compression, 3)
+
+
+class TestLoadHDF5(unittest.TestCase):
+
+    def setUp(self):
+        _, path = tempfile.mkstemp()
+        self.temp_file_path = path
+        with h5py.File(path, 'w'):
+            pass
```
Comment (beam2d) [inline tests/chainer_tests/serializers_tests/test_hdf5.py]: Is this `with` statement necessary?

File: tests/chainer_tests/serializers_tests/test_hdf5.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+    def tearDown(self):
+        if hasattr(self, 'temp_file_path'):
+            os.remove(self.temp_file_path)
+
+    def test_save(self):
```
Comment (beam2d) [inline tests/chainer_tests/serializers_tests/test_hdf5.py]: test_load?

**Comment by beam2d** [comment]
LGTM except two comments.

File: tests/chainer_tests/serializers_tests/test_hdf5.py (hunk 1 above)
Comment (unnonouno) [inline tests/chainer_tests/serializers_tests/test_hdf5.py]: OK, I'll use only `File` to make an empty file.

File: tests/chainer_tests/serializers_tests/test_hdf5.py (hunk 2 above)
Comment (okuta) [inline tests/chainer_tests/serializers_tests/test_hdf5.py]: `test_save` is collect.

**Pull request merged**